src/features/pricing/berthPricing/__tests__/BerthPricing.tests.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import BerthPricing, { BerthPricingProps } from '../BerthPricing';
+import { data } from '../__fixtures__/data';
+
+describe('BerthPricing', () => {
+  const initialProps: BerthPricingProps = { data, loading: false, openModal: jest.fn() };
+
+  const getWrapper = (props: Partial<BerthPricingProps> = {}) => shallow(<BerthPricing {...initialProps} {...props} />);
+
+  beforeEach(() => {
```

Comment:
I think we don't need this, there is no mocks to reset?

Reply:
There's `openModal: jest.fn()` In the `initialProps`, it wasn't tested though, I have added tests to simulate clicking on edit buttons.